### PR TITLE
feat: Add protocol statistic 

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

The component is ready but it had hardcoded values bc we need to merge the SG... When the SG is ready I will add a little improvements and will be ready

### CODE REVIEW

**Review by Agupane — COMMENTED** [review]
Ok that was fast, great work!
I deployed the new subgraph, you can go ahead with the connection

File: i18n/translations.en.json [hunk 1]
```diff
@@ -60,6 +60,17 @@
         "description": "Third-party Badges are a blockchain certification service that leverages TheBadge's platform, allowing third-party entities to issue their own certificates. These badges are backed by the issuing entity, providing added legitimacy to the certification. By utilizing TheBadge's platform, these entities can ensure secure storage and verification of their certification data, providing a reliable certification process. Third-party Badges allow individuals to showcase their achievements in a trusted ecosystem, enhancing their credibility in the wider community. ",
         "documentation": "thebadge-documentation/what-is-a-badge/badge-categories/third-party-badges"
       }
+    },
+    "statistics": {
+      "title": "Protocol statistics",
+      "created": "BADGES CREATED",
+      "types": "TYPE OF BADGES",
+      "challenged": "BADGES CHALLENGED",
+      "owners": "BADGES\nOWNERS",
```
Comment (ramabit) [inline i18n/translations.en.json:69]: typo, should say: BADGE\nOWNERS or simply OWNERS
btw I would put it not all uppercase in this file, but modify that with css

File: i18n/translations.en.json [hunk 2]
```diff
@@ -60,6 +60,17 @@
...
+      "creators": "BADGES\nCREATORS",
```
Comment (ramabit) [inline i18n/translations.en.json:70]: typo, should say: BADGE\nCREATORS or simply CREATORS
btw I would put it not all uppercase in this file, but modify that with css

File: i18n/translations.en.json [hunk 3]
```diff
@@ -60,6 +60,17 @@
...
+      "curators": "BADGES CURATORS"
```
Comment (ramabit) [inline i18n/translations.en.json:71]: typo, should say: BADGE\nCURATORS or simply CURATORS
btw I would put it not all uppercase in this file, but modify that with css

**Review by ramabit — CHANGES_REQUESTED** [review]

**Review by Agupane — APPROVED** [review]

**Review by ramabit — APPROVED** [review]

**Pull request merged**